src/Sulu/Bundle/ContentBundle/Collaboration/Collaboration.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Collaboration;
+
+/**
+ * Represents a user, which is currently editing an entity.
+ */
+class Collaboration
```

Comment:
Good point... When thinking about it, this should maybe actually go into an own Bundle, so that it can also be reused without the ContentBundle at all... @wachterjohannes @chirimoya WDYT?